.github/workflows/test-suite.yml

```diff
@@ -78,8 +76,8 @@ jobs:
     name: release-tests-ubuntu
     needs: [check-labels]
     if: needs.check-labels.outputs.skip_ci != 'true'
-    # Use self-hosted runners only on the sigp repo.
-    runs-on: ${{ github.repository == 'sigp/lighthouse' && fromJson('["self-hosted", "linux", "CI", "large"]') || 'ubuntu-latest'  }}
+    # Moved to warpbuild runner
+    runs-on: warp-ubuntu-latest-x64-8x
```

Comment:
is there a way to detect if warp is available on the fork? 
the condition before was to maintain support on running runners on fork as well, i.e. run on github-hosted instance if self hosted runners aren't available.